config/manager/kustomization.yaml

```diff
@@ -53,3 +53,13 @@ configMapGenerator:
 - name: default-repos-rhel-91
   files:
   - repositories/rhel-91.json
+
+vars:
```

Comment:
There is a default value?

Reply:
What do you mean? It gets the value from the image field of the manager container in the deployment spec. Why would it have a default value?

Reply:
I thought the user setting it as part of the deployment

Reply:
Again, I don't understand what you mean. The code needs to know the container image to use for the ISO repackaging job. For that, we added a new configuration value that gets its value from the Environment Variables. We agreed that we will put the executable inside the same container as the manager and httpapi. So, we don't want to hard-code this value because during development the value changes. But, we don't want to change it manually. Instead, I'm using Kustomize's `vars` mechanism